screeps_console/interactive.py

```diff
@@ -278,25 +278,23 @@ def disconnect(self):
             self.proc = False
 
     def onUpdate(self, data):
-
         # If we lose the connection to the remote system close the console.
-        if data.startswith('### closed ###'):
+        if data.startswith(b'### closed ###'):
```

Comment:
What's the point of turning these into "bytes" instead of just strings? You do it in some other areas as well.

Reply:
`data` is a byte array, in Python 3. `startswith` will throw on mismatched types.

Reply:
Ah, that makes sense.